common/utils/src/main/resources/error/error-conditions.json

```diff
@@ -2941,6 +2941,23 @@
     ],
     "sqlState" : "22029"
   },
+  "INVALID_VARIABLE_DECLARATION": {
+    "message" : [
+      "Invalid variable declaration."
+    ],
+    "subClass" : {
+      "NOT_ALLOWED_IN_SCOPE" : {
+        "message": [
+          "Variable declaration is not allowed in this scope."
+        ]
+      },
+      "ONLY_AT_BEGINNING" : {
+        "message": [
+          "Variable declaration is only possible at the beginning of the BEGIN END compound."
```

Comment:
nit: you can remove BEGIN END part from the message, compound is literally defined by BEGIN and END.